power-the-polls-form/src/components/ptp-info-jurisdiction/MichiganAdditionalInfoForm.tsx

```diff
@@ -0,0 +1,210 @@
+import { Fragment, h } from "@stencil/core";
+import { FunctionalComponent } from "@stencil/router/dist/types/stencil.core";
+
+import { PtpFormData } from "../../util";
+import { submitToActionKit } from "../power-the-polls-form/ActionKit";
+
+const RadioOptions: FunctionalComponent<{ choices: string[], inputName: string }> =
+   ( { choices, inputName } ) => (
+      <Fragment>
+         {choices.map( x => <label key={x} class="checkbox"><input type="radio" name={inputName} value={x} required />{" "}{x}</label> )}
+      </Fragment>
+   );
+
+const SelectOptions: FunctionalComponent<{ choices: string[], inputName: string }> =
+   ( { choices, inputName } ) => (
+      <select name={inputName} required>
+         <option value=""></option>
+         {choices.map( x => <option value={x}>{x}</option> )}
+      </select>
+   );
+
+/**
+ * Michigan requires additional info be collected.
+ * @see
```

Comment:
Does this link to anything?

Reply:
I do not know...

Reply:
what does @see suppose to do ?

Reply:
Reference another file. I was probably going to, or used to, reference the component where this was used.